src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/group/RabbitMessageGroupBatchSender.kt

```diff
@@ -44,7 +51,23 @@ class RabbitMessageGroupBatchSender(
             MESSAGE_GROUP_PUBLISH_TOTAL,
             MESSAGE_GROUP_SEQUENCE_PUBLISH
         )
-        super.send(value)
+        val batchBuilder = MessageGroupBatch.newBuilder()
```

Comment:
Execute a lightweight check before rebuilding the batch to avoid redundant operations.

Reply:
done